src/Plotsrecipe.jl

```diff
@@ -0,0 +1,40 @@
+using RecipesBase
```

Comment:
You either have to make this conditional like the compose check, or add RecipesBase to the REQUIRE file